pwa-devdocs/src/venia-pwa-concept/project-structure/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Venia project structure
+---
+
+This topic is an overview of the [`venia-concept`][] project structure.
+It provides overview information about important directories and files to help you understand the different pieces of the project.
+
+The Venia PWA storefront isn't a traditional Magento Theme like the Blank and Luma themes and therefore differs from the traditional [Magento theme structure][].
+It's not part of a Magento code base but a separate instance that communicates with Magento through the [UPWARD][] middleware.
+
+## Root directory files
+
+In addition to the NPM packages.json and Venia specific validation and testing files, the root directory contains the following important files:
+
+[`.env.dist`][]
+: A sample configuration file that defines important environment variables.
+  Copy this file into a new `.env` file to use the project default values.
+
+  The first change to make here is adjusting the `MAGENTO_BACKEND_URL` variable to the fully qualified URL of a Magento 2.3 instance.
+  Read more on the [Venia setup][] page.
+
+[`deployVeniaSampleData.sh`][]
+: This file helps you to [Install Venia sample data][] into your Magento installation.
+  Copy and execute this file in a Magento instance to install the sample data.
+
+[`server.js`][]
+: A node script that runs the UPWARD staging server when the `npm run stage:venia` command is used.
+
+[`validate-environment.js`][]
+: A node script that validates the `.env` environment file.
+
+[`validate-queries.js`][]
+: validates that all `.graphql` query files in the project are valid for the GraphQL schema of the Magento backend instance specified in `.env`.
+
+[`venia-upward.yml`][]
+: The [UPWARD server specification][] for the Venia PWA storefront.
+  This file describes the server behavior for the middle tier service between the PWA and Magento.
+
+[`webpack.config.js`][]
+
```

Comment:
Not sure if this matters but line 38 is blank here. All the other entries in this section immediately follow on the next line.